### ISSUE TITLE
3 Months Before Event: Decide on Speakers

### ISSUE THREAD

**Issue by yigakpoa** [chaoss/community#613]
- Decide on Speakers 
- To be overseen by (Content Coordinator + Planning Committee)

PS. There's no Content Coordinator at the time of creating this issue so maybe Dawn & Elizabeth can take a look?

**Comment by geekygirldawn** [comment]
Since we don't have much time to prep for this and we're just finishing CHAOSScon EU, we decided not to have a CFP, but we do need a content coordinator who will need to build the schedule and make sure that we have activities.

We've talked about having interactive sessions where people can work together on some metrics definitions and insight guides, but this will require some prep from the content coordinator or someone they delegate to.

We also talked about doing lightning talks. 

cc: @GeorgLink
